src/module.ts

```diff
@@ -100,6 +101,11 @@ export default defineNuxtModule<ModuleOptions>({
       }
     }
 
+    nuxt.options.alias = {
+      ...nuxt.options.alias,
+      '#nuxt-open-fetch-schemas': join(nuxt.options.buildDir, 'types', moduleName, 'schemas'),
```

Comment:
What do you think about '#open-fetch-schemas'?

Reply:
i'm really bad at naming so i'm good with any suggestion :joy:

Reply:
Same problem)
I recently removed "nuxt-" prefixes from all imports, realised that it is obvious that the imports/aliases will not be used in the Next.js 😅